docs/changelog/1075.md

```diff
@@ -0,0 +1 @@
+- Removed deprecated fortran bindings: `precicef_ongoing()` `precicef_write_data_required()` `precicef_read_data_available()` `precicef_action_required()`  
```

Comment:
```suggestion
- Removed deprecated fortran bindings: `precicef_ongoing()`, `precicef_write_data_required()` ,`precicef_read_data_available()`, and `precicef_action_required()`. One-to-one replacement functions are `precicef_is_ongoing()`, `precicef_is_write_data_required()`, `precicef_is_read_data_available()`, and `precicef_is_action_required()`. 
```
Please double-check that I got the names right.

Reply:
Thank you. It's `precicef_is_coupling_ongoing` instead of is `precicef_is_ongoing` and everything now included in the next commit.